ls1intum/Apollon_standalone
# `Documentation`: Update documentation of Standalone

### Motivation and Context
This PR simply updates the README of Standalone project.

### Steps for Testing
1. Goto the [URL](https://github.com/ls1intum/Apollon_standalone/tree/chores/update-readme).
2. Observe the updated readme of the project
